src/styles/global.css

```
@@ -209,6 +209,188 @@ body[data-mobile-nav-open] {
+.post-body blockquote > :last-child {
+	margin-bottom: 0;
+}
+
+.post-body code {
+	font-family: 'IBM Plex Mono', monospace;
+	font-size: 0.92em;
+	padding: 0.08rem 0.35rem;
+	border: 1px solid var(--border);
+	background: var(--surface);
+	border-radius: 0.15rem;
+	word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)